src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -850,7 +873,39 @@ public void integrationTest() throws Exception {
 
     // Start container
     sut.startContainer(id);
-
+    
+    
+    // Copy files to container
+    final String dockerDirectory = Resources.getResource("dockerSslDirectory").getPath();
+    try {
+    	sut.copyToContainer(Paths.get(dockerDirectory), id, "/tmp");
+	} catch (Exception e) {
+		fail("error to copy files to container");
```

Comment:
ah you're right

On Thu, Nov 12, 2015 at 3:52 AM, walbus <notifications@github.com> wrote:

> In src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java
> <https://github.com/spotify/docker-client/pull/234#discussion_r44633154>:
>
> > @@ -850,7 +873,39 @@ public void integrationTest() throws Exception {
> >
> >      // Start container
> >      sut.startContainer(id);
> > -
> > +
> > +
> > +    // Copy files to container
> > +    final String dockerDirectory = Resources.getResource("dockerSslDirectory").getPath();
> > +    try {
> > +    	sut.copyToContainer(Paths.get(dockerDirectory), id, "/tmp");
> > +	} catch (Exception e) {
> > +		fail("error to copy files to container");
>
> Also version 1.20
>
> http://docs.docker.com/engine/reference/api/docker_remote_api_v1.20/#extract-an-archive-of-files-or-folders-to-a-directory-in-a-container
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/234/files#r44633154>.
>



-- 
David Xia
Software Engineer
dxia@spotify.com